lesson-10/src/main/java/ru/otus/lesson/MyIoc.java

```diff
@@ -0,0 +1,12 @@
+package ru.otus.lesson;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Proxy;
+
+public class MyIoc {
+    public static ru.otus.lesson.Logging createMyClass() throws ClassNotFoundException {
```

Comment:
Лучше используйте import, а не полные имена пакетов.
В других местах тоже.

```suggestion
    public static Logging createMyClass() throws ClassNotFoundException {
```

Reply:
Поправил